packages/web-twig/src/Resources/components/Tooltip/stories/TooltipDismissibleViaJS.twig

```diff
@@ -0,0 +1,12 @@
+<p>
+  Saves data to local storage.
+</p>
+
+<TooltipWrapper>
+  <Button id="my-dismissible-button" aria-describedby="my-dismissible-tooltip2">I have a tooltip 😎</Button>
+  <Tooltip id="my-dismissible-tooltip2" placement="right" isDismissible UNSAFE_className="is-hidden">
+      Close me
+  </Tooltip>
+</TooltipWrapper>
+
+{{ encore_entry_script_tags('tooltipDismissibleViaJS') }}
```

Comment:
again an empty line at the bottom of the file, please

Reply:
space added